docs/types/operators.rst

```diff
@@ -26,6 +26,18 @@ except for comparison operators where the result is always ``bool``.
 The operators ``**`` (exponentiation), ``<<``  and ``>>`` use the type of the
 left operand for the operation and the result.
 
+.. warning::
```

Comment:
Some suggestions:
 - please check how we call the operator in other parts of the documentation and at least also add `? ... : ...` as an alternative here, so that everyone knows what we are talking about
 - "literal" is more strict than what we want here. I think we use the term rational constant or something, please check.

At this point in the documentation, I think a "note" is better then a warning - we don't use a warning for `**` either.